common/types/types.go

```diff
@@ -768,6 +768,19 @@ func ProtoAsType(t *celpb.Type) (*Type, error) {
 	}
 }
 
+// TypeToProto converts from a CEL-native type representation to canonical CEL celpb.Type protobuf type.
```

Comment:
Would you mind moving this logic to the `common/decls/decls.go` file? There's some precedent for interop between the alpha and canonical protobuf types in that file.

Reply:
This is the inverse of the preceding function in this file -- are you sure you'd prefer it in decls/decls.go? There aren't any dependency changes so not sure I follow your comment

Reply:
My apologies, I had recently refactored some of the methods for type conversion in the `decls.go` and forgotten about the ones in `types.go` as well. This is fine.